sceptre/scipool/templates/sc-enduser-iam.yaml

```diff
@@ -10,6 +10,7 @@ Resources:
       RoleName: ServiceCatalogEndusers
       ManagedPolicyArns:
         - arn:aws:iam::aws:policy/AWSServiceCatalogEndUserFullAccess
+        - arn:aws:iam::aws:policy/CloudWatchLogsReadOnlyAccess
```

Comment:
Seems fine- people need to be conscious of not logging anything sensitive anyway